### ISSUE TITLE
Can't change the colors

### ISSUE THREAD

**Issue by silviahisham** [jaydeepw/poly-picker#81]
I use this code to change colors, but it doesn't work, neither the tab indicator or the camera button changes their color 

    Config config = new Config.Builder()
                .setTabBackgroundColor(R.color.white)    // set tab background color. Default white.
                .setTabSelectionIndicatorColor((R.color.primaryAppColor)
                .setCameraButtonColor(R.color.green)
                .setSelectionLimit(4)    // set photo selection limit. Default unlimited selection.
                .build();

**Comment by nishantjain91** [comment]
For camera button color, the library uses porter duff darken mode which will only work if the valueof your color is smaller than the default color.

**Comment by silviahisham** [comment]
but what about the tab indicators? they're always blue, I can't change them.

**Comment by mmanishh** [comment]
Yes, i have the same issue working on . Crashed when i tried to  change tab colors
